app/modules/core/models/settings.py

```diff
@@ -44,6 +45,57 @@ def for_site(cls, site):
         return instance
 
 
+class ThemeChoices(models.TextChoices):
+    NHS_DARK_PINK = "nhs-dark-pink", "NHS Dark Pink"
+    NHS_DARK_GREY = "nhs-dark-grey", "NHS Dark Grey"
+    NHS_PALE_GREY = "nhs-pale-grey", "NHS Pale Grey"
+    NHS_LIGHT_BLUE = "nhs-light-blue", "NHS Light Blue"
+
+
+@register_setting
+class HomePageBannerSettings(CachedSetting):
```

Comment:
There's a part of me that really wants to collapse these into a shared parent class, but it's not necessary.